framework/vst/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@ if (NOT VST3_SDK_PATH)
     message(FATAL_ERROR "VST3_SDK path is not setted")
```

Comment:
Typo, "setted" is not an English word ;-)

Reply:
fixed. Thanks!)